src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/ObjectWriter.cs

```diff
@@ -341,6 +476,9 @@ public static void EmitObject(string objectFilePath, IEnumerable<DependencyNode>
                         // Emit symbol definitions if necessary
                         objectWriter.EmitSymbolDefinition(i);
 
+                        // Emit CFI codes for the given offset.
```

Comment:
This change is adding 3 more hash table lookups per emitted byte of code ... I am starting to worry about how much stuff the emitter is doing per single byte.